### PR TITLE
set rounds as the redirectAfterSignIn if user already has an account

### PR DESCRIPTION
**Pull request by muganwas**

### CODE REVIEW

File: src/components/header/HeaderOld.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ function Header ({ user, users, round, setUser, setIsShowingSignInDialog, redire
                     setUser(user)
                 }
                 console.log('redirectAfterSignIn', redirectAfterSignIn);
-                redirect()
+                redirect();
```
Comment (choshun) [inline src/components/header/HeaderOld.js:109]: ty!!

File: src/components/play/PlayUI.js [hunk 2]
```diff
@@ -765,7 +765,7 @@ class PlayUI extends Component {
 
     addLayer (layer, order, shouldAnimate = true) {
         // console.log('addLayer', layer);
```
Comment (choshun) [inline src/components/play/PlayUI.js:767]: can you remove this?

File: src/components/play/PlayUI.js [hunk 3]
```diff
@@ -679,8 +679,8 @@ class PlayUI extends Component {
     loadPattern (userId, id, order) {
         // console.log('load pattern', id);
         // console.time('loadPattern')
-        //this.props.dispatch({ type: SET_CURRENT_SEQUENCE_PATTERN, payload: { value: order } })
-        const pattern = _.find(this.props.round.userPatterns[userId].patterns, { id })
+        // this.props.dispatch({ type: SET_CURRENT_SEQUENCE_PATTERN, payload: { value: order } })
```
[the remark this answers is not in the document]
Comment (muganwas) [reply src/components/play/PlayUI.js:682]: Sure.

File: src/components/play/PlayUI.js (hunk 2 above)
Comment (muganwas) [reply src/components/play/PlayUI.js:767]: Alright.

File: src/components/header/Header.js [hunk 4]
```diff
@@ -85,6 +85,7 @@ class Header extends Component {
 					//console.log('CustomSamples', CustomSamples.samples);
```
[the remark this answers is not in the document]
Comment (muganwas) [reply src/components/header/Header.js:85]: Sure

**Pull request closed without merging**